readme.md

```diff
@@ -37,7 +38,7 @@ Your Answer:
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 Your Answer:
-```text
+```SQL are table databases where data is represented with rows and columns.  SQL = MongoDB
 
 ```
```

Comment:
SQL != MongoDB,
MongoDb == NoSQL

SQL databases are built around the idea of tables, rows (entries), and columns (attributes). SQL databases are optimized for relational data, and support relating data through JOIN operations and Foreign Keys. SQL databases have a rigid schema, which describes the structure of the tables and columns.

NoSQL databases often lack a schema, and allow us to insert any documents (similar to JSON objects) we want into the databases collections (related groups of objects). NoSQL can be faster for some cases, but slower when dealing with relational data.